sdk/eventgrid/eventgrid/src/predicates.ts

```diff
@@ -440,11 +440,13 @@ export function isSystemEvent<T extends KnownSystemEventTypes>(
  * @param eventType The type of system event to check for, e.g., "Microsoft.AppConfiguration.KeyValueDeleted"
  * @param event The event to test.
  */
+// eslint-disable-next-line no-redeclare
```

Comment:
Do you know what needs to be done to get this error to go away?  My goal was to follow the general typescript overload pattern here, but it's possible I screwed something up.

Reply:
This stuck out to me as well.

Reply:
@ellismg never mind, I should have used a different rule as explained here https://github.com/Azure/azure-sdk-for-js/pull/12770#discussion_r540503660. I am reverting this change.